setup.py

```diff
@@ -178,6 +178,8 @@ def version(self):
 extra_reqs = {
     'test': ['pytest', 'pytest-cov', 'numpy>=1.4.1', 'packaging']
 }
+if os.getenv('NUMPY') == '0':
+    extra_reqs['test'].remove('numpy>=1.4.1')
```

Comment:
No, not unless they installed `shapely[vectorized]` or `shapely[all]`. `pip install shapely` won't get any of the extra requirements.